pkg/controllers/dashboard/helm/repo.go

```diff
@@ -32,10 +34,14 @@ var (
 	interval = 5 * time.Minute
 )
 
+type configMapGetter interface {
+	Get(namespace, name string, options metav1.GetOptions) (*corev1.ConfigMap, error)
+}
+
 type repoHandler struct {
 	secrets      corev1controllers.SecretCache
 	clusterRepos catalogcontrollers.ClusterRepoController
-	configMaps   corev1controllers.ConfigMapClient
+	configMaps   configMapGetter
```

Comment:
ConfigMapClient is already an interface so this isn't needed. There is also existing work around generating mocks for the interfaces used in the clients.